### PR TITLE
OCP: Added control response for SI group

### PR DESCRIPTION
**Pull request by Vincent056**
Added control response for SI-4(18), SI-4(20), SI-4(22)

### CODE REVIEW

File: controls/nist_ocp4.yml [hunk 1]
```diff
@@ -15699,13 +15698,15 @@ controls:
...
   rules: []
 - id: SI-4(22)
-  status: pending
+  status: automated
   notes: |-
-    A control response for SI-4(20) is planned. Engineering progress can be
-    tracked via:
+    The customer is responsible for ensuring that NetworkPolicies
+    are configured in all non-openshift namespaces to allow traffic
+    only in the namespace and not across namespaces[1]
```
Comment (jhrozek) [inline controls/nist_ocp4.yml:15705]: I think we should reword this a little and explain that while OCP doesn't provide a way to audit the services per se, explicitly setting NetworkPolicies would help. They might still need another way to make the service approved from their end (inventory, trusting only certain CAs...)

File: controls/nist_ocp4.yml [hunk 2]
```diff
@@ -15649,12 +15649,10 @@ controls:
...
 - id: SI-4(18)
-  status: pending
+  status: not applicable
   notes: |-
-    A control response for SI-4(18) is planned. Engineering progress can be
-    tracked via:
-
-    https://issues.redhat.com/browse/CMP-537
+    OpenShift Container Platform does not provides firewall gateway.
+    A third party firewall should be used for traffic analysis.
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply controls/nist_ocp4.yml:15655]: Just got answer from them about this one, so they are using palo alto firewall for that purpose, so I guess this is good here?

File: controls/nist_ocp4.yml (hunk 1 above)
Comment (Vincent056) [reply controls/nist_ocp4.yml:15705]: I am wondering how is `inventory, trusting only certain CAs` helping this case?

File: controls/nist_ocp4.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhrozek) [reply controls/nist_ocp4.yml:15655]: Ah, then leave the reply as it is. I don't think we should point to a particular service (sounds too much like an ad..)

File: controls/nist_ocp4.yml (hunk 1 above)
Comment (jhrozek) [reply controls/nist_ocp4.yml]: So inventory is just a list of the approved services. A way to match the network policies to the outside. Trusting only certain CAs is probably only helpful for routes (see e.g. https://docs.openshift.com/container-platform/4.10/networking/routes/secured-routes.html) so they check that only the CA certs that the org should be trusting are set for routes. In-cluster communication is done using self-signed certs anyway.

**Review by jhrozek — APPROVED** [review]
LGTM

**Pull request merged**